presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -73,14 +73,14 @@ public MySqlClient(JdbcConnectorId connectorId, BaseJdbcConfig config, MySqlConf
     }
 
     @Override
-    public Statement getStatement(Connection connection)
+    public PreparedStatement getPreparedStatement(Connection connection, String sql)
             throws SQLException
     {
-        Statement statement = connection.createStatement();
-        if (statement.isWrapperFor(com.mysql.jdbc.Statement.class)) {
```

Comment:
We can actually keep this is as `com.mysql.jdbc.Statement`, since that is the MySQL-specific interface that declares `enableStreamingResults()`.